ikarus/io/resultevaluators.hh

```diff
@@ -75,27 +104,121 @@ struct PrincipalStress
 {
   /**
    * \brief Calculate the result quantity (principal stress)
-   * \param resultArray EigenMatrix containing the stress state
+   * \param resultArray EigenVector containing the stress state in Voigt notation
    * \param comp component of result
    * \return principal stress
    */
-  double operator()(const auto& resultArray, const int comp) const {
+  double operator()(const auto& resultArray, [[maybe_unused]] const auto& pos, [[maybe_unused]] const auto& fe,
+                    const int comp) const {
     auto mat = fromVoigt(resultArray, false);
     Eigen::SelfAdjointEigenSolver<decltype(mat)> eigensolver(mat, Eigen::EigenvaluesOnly);
     return eigensolver.eigenvalues()[dim - 1 - comp];
```

Comment:
Sorry for this change in old code but this would be nicer with eigensolver.eigenvalues().reverse()[comp] because then no index fiddling is necessary.